test_scripts/Polices/build_options/016_ATF_P_TC_Timeout_Countdown_Start_PTU_Response_HTTP.lua

```diff
@@ -120,11 +114,11 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
     return true
   end
 
-  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP", fileType = "JSON"})
+  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP", fileType = "BINARY"})
```

Comment:
@istoimenova, Please add Timeout(time_wait). Because now test fails 
By the way commonTestCases:DelayedExp(time_wait) can be removed